apigateway/helm/values.yaml

```diff
@@ -355,8 +358,9 @@ apigw:
   #   allowEdit: false
   #   properties:
   #     location: apigw-config.yml
-  extraConfigSource: {}
-  
+  extraConfigSource: []
```

Comment:
Not sure - does this work in the template. Name should be changed to plural extraConfigSources if array.

Reply:
changed name to extraConfigSources